Expensify/App
# Revert test changes

### Details
Reverts the following dummy PRs:

- https://github.com/Expensify/Expensify.cash/pull/3303
- https://github.com/Expensify/Expensify.cash/pull/3304
- https://github.com/Expensify/Expensify.cash/pull/3307
- https://github.com/Expensify/Expensify.cash/pull/3315
- https://github.com/Expensify/Expensify.cash/pull/3319
- https://github.com/Expensify/Expensify.cash/pull/3324
- https://github.com/Expensify/Expensify.cash/pull/3328
- https://github.com/Expensify/Expensify.cash/pull/3331
- https://github.com/Expensify/Expensify.cash/pull/3336

### Fixed Issues
See https://github.com/Expensify/Expensify/issues/155229#issuecomment-853515530 for more context

### Tests / QA Steps
1. Run the app
1. Make sure you don't see any incorrect or otherwise wonky colors
1. Go to `Settings` -> `About` and verify that you do not see the `Dummy change!` text.
